src/SmartComponents/BaselinesPage/CreateBaselineModal/CreateBaselineModal.js

```diff
@@ -48,13 +48,37 @@ export class CreateBaselineModal extends Component {
         };
     }
 
+    buildSystemColumns = (originalColumns) => {
+        const { permissions } = this.props;
+        let columns = [ ...originalColumns ];
+        let index = columns.findIndex(column => column.key === 'display_name' || column.key === 'display_selected_hsp');
```

Comment:
Observation: can use `map` instead of in-place change by index. Hint: use spread syntax to create a copy of an object.